libsplinter/src/auth/oauth/mod.rs

```diff
@@ -194,6 +203,22 @@ impl OAuthClient {
             pending_authorization.client_redirect_url,
         )))
     }
+
+    /// Uses an access token to retrieve the user's identity from the OAuth provider.
+    ///
+    /// # Arguments
+    ///
+    /// * `access_token` - The access token required to authorize the request for the identity
+    pub fn get_identity(&self, access_token: String) -> Result<String, OAuthClientError> {
```

Comment:
The identity should be added to the tokens struct (which should probably be renamed now) instead of being returned with a separate method.